reference/docs/conversion/sakai_11_mysql_conversion.sql

```diff
@@ -492,118 +492,194 @@ UPDATE MFR_PERMISSION_LEVEL_T SET IDENTIFY_ANON_AUTHORS = 1 WHERE NAME = 'Owner'
 -- SAM-2627
 ALTER TABLE SAM_GRADINGATTACHMENT_T ADD COLUMN `ASSESSMENTGRADINGID` BIGINT(20) NULL AFTER `ITEMGRADINGID`;
 
--- SAK-25544 - Quartz updated to 2.2.1
-
+-- SAK-25544/SAK-31365 - Quartz updated to 2.2.1
+--
 -- drop tables that are no longer used
 --
-DROP TABLE QRTZ_JOB_LISTENERS;
-DROP TABLE QRTZ_TRIGGER_LISTENERS;
+DROP TABLE qrtz_job_listeners;
+DROP TABLE qrtz_trigger_listeners;
```

Comment:
Sorry I didn't catch this earlier .... but table names need to stay in proper case